ui/src/app/shared/requirements/form/requirements.form.component.ts

```diff
@@ -120,6 +128,9 @@ export class RequirementsFormComponent implements OnInit {
             case 'volume':
                 this.newRequirement.name = this.getVolumeName();
                 break;
+            case 'os-architecture':
```

Comment:
could be interesting to create a const variable in this class in order to not mispell the requirement type. It's only used in this class I think and in the template